docs/user-guides/dashboard-as-code.md

```diff
@@ -37,23 +37,23 @@ See the [CUE documentation](https://cuelang.org/docs/concepts/packages/) for mor
 
 #### 2. Retrieve the CUE sources from Perses
 
-Ideally we should rely on a native dependency management here, but since it's not yet available for CUE as already mentionned, we provide in the meantime a dedicated CLI command `dac setup` in order to add the CUE sources from Perses as external dependencies to your repo:
+Ideally we should rely on a native dependency management here, but since it's not yet available for CUE as already mentioned, we provide in the meantime a dedicated CLI command `dac setup` in order to add the CUE sources from Perses as external dependencies to your repo:
 
 ```
 percli dac setup --version 0.42.1
 ```
 
-You can omit the version flag if your are connected to a Perses server (it will retrieve its version). Ottherwise, unless you have a specific case, better to pass the latest version available.
+You can omit the version flag if you are connected to a Perses server (it will retrieve its version). Otherwise, unless you have a specific case, better to pass the latest version available.
 
-## Develop dashboards
+### Develop dashboards
 
 You are now fully ready to start developping dashboards as code!
 
 It's first strongly recommended to ramp up on CUE if you are not familiar with this technology. For this have a look at:
 - The [official website](https://cuelang.org/) of Cuelang.
 - [Cuetorials](https://cuetorials.com/), a 3rd party source of information that is a very good complement.
 
-Then, you can check an example of DaC usage [here](../../internal/test/dac/input.cue). This example is heavily relying on the DaC utilities we provide. To get a deeper understanding of these libs and how to use them, the best thing to do for now is to check directly their source code.
+Then, you can check an example of DaC usage [here](../../internal/test/dac/input.cue). This example is heavily relying on the DaC utilities we provide. To get a deeper understanding of the Go SDK and how to use it, the best thing to do for now is to check directly its source code.
 
 Anytime you want to build the final dashboard definition (i.e Perses dashboard in JSON or YAML format) corresponding to your as-code definition, you can use the `dac build` command, as the following:
```

Comment:
the `dac build` part here could be removed and factorize with the Go one below, as it's exactly the same. Only the example differ, and you could put an example of each in the same code block